app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -46,17 +46,17 @@
   <dimen name="submitted_answer_exploration_view_margin_start">208dp</dimen>
   <dimen name="text_input_interaction_item_conversation_view_margin_start">208dp</dimen>
 
-  <dimen name="continue_interaction_item_question_view_margin_end">192dp</dimen>
-  <dimen name="continue_interaction_item_question_view_margin_start">192dp</dimen>
+  <dimen name="continue_interaction_item_question_view_padding_end">192dp</dimen>
```

Comment:
Why is this getting changed? This PR is not using this change anywhere.

Reply:
Done.